core/src/main/scala/monocle/Plated.scala

```diff
@@ -0,0 +1,84 @@
+package monocle
+
+import scalaz._
+import scalaz.std.stream._
+
+/**
+  * [[Plated]] is a type-class for types which can extract their immediate
+  * self-similar children.
+  *
+  * @tparam A the parent and child type of a [[Plated]]
+  */
+abstract class Plated[A] extends Serializable { self =>
+  def plate: Traversal[A, A]
+}
+
+object Plated {
+  def plate[A](implicit P: Plated[A]): Traversal[A, A] = P.plate
+  @inline def children[A: Plated](a: A): List[A] = plate[A].getAll(a)
+  def universe[A: Plated](a: A): Stream[A] = {
+    val fold = plate[A].asFold
+    def go(b: A): Stream[A] = b #:: fold.foldMap[Stream[A]](go)(b)
+    go(a)
+  }
+  def rewrite[A: Plated](f: A => Option[A])(a: A): A =
+    rewriteOf(plate[A].asSetter)(f)(a)
+  def rewriteOf[A](l: Setter[A, A])(f: A => Option[A])(a: A): A = {
+    def go(b: A): A = {
+      val c = transformOf(l)(go)(b)
+      f(c).fold(c)(go)
+    }
+    go(a)
+  }
+  def transform[A: Plated](f: A => A)(a: A): A =
+    transformOf(plate[A].asSetter)(f)(a)
+  def transformOf[A](l: Setter[A, A])(f: A => A)(a: A): A =
+    l.modify(b => transformOf(l)(f)(f(b)))(a)
+
+  implicit def freePlated[S[_]: Traverse, A]: Plated[Free[S, A]] = new Plated[Free[S, A]] {
+    def plate: Traversal[Free[S, A], Free[S, A]] = new Traversal[Free[S, A], Free[S, A]] {
+      def modifyF[F[_]: Applicative](f: Free[S, A] => F[Free[S, A]])(s: Free[S, A]): F[Free[S, A]] =
+        s.resume.fold(
+          as => Applicative[F].map(Traverse[S].traverse(as)(f)) {
+            // Free.roll does not exist in 7.1
+            Free.liftF(_).flatMap(identity)
+          },
+          x => Applicative[F].point(Free.point(x))
+        )
+    }
+  }
+
+  implicit def listPlated[A]: Plated[List[A]] = new Plated[List[A]] {
```

Comment:
would you mind to move the instances of `Plated` to `monocle.std.x`, in that case `monocle.std.list`